chainstate/src/detail/transaction_verifier/mod.rs

```diff
@@ -571,6 +639,21 @@ impl<'a, S: BlockchainStorageRead> TransactionVerifier<'a, S> {
         Ok(())
     }
 
+    fn unregister_token_issuance(
```

Comment:
Please put all register/unregister functions together so that they can be compared (including undo, and whatever else related). Putting them in utils isn't the worst idea, but I don't think that's easily achived because they have to do with the cache. So please just put them above one another somewhere here.

Reply:
If you agreed with parameter `tokens_cache: &mut BTreeMap<TokenId, CachedTokensOperation>,`, then we can move all these functions into utils

Reply:
Go for it. Preferably put them all in the tokens submodule, not the generic utils.

Reply:
Done